pkg/apic/apiserver/models/api/v1/resourceinstance.go

```diff
@@ -1,13 +1,64 @@
 package v1
 
+import (
+	"bytes"
+	"encoding/json"
+)
+
 // ResourceInstance API Server generic resource structure.
 type ResourceInstance struct {
 	ResourceMeta
-	// GENERATE: The following code has been modified after code generation
-	//  	Owner struct{} `json:"owner"`
-	Owner *struct{} `json:"owner,omitempty"`
+
+	Owner interface{} `json:"owner,omitempty"`
```

Comment:
You mentioned it is modified after code gen. Does that mean we have to change it every time we run the code gen?

Reply:
That was the old codegen but our scripts would handle that.  Now its an interface and there are no additional mods needed for Owner